.changeset/sweet-ties-juggle.md

```diff
@@ -0,0 +1,7 @@
+---
+'@openfn/language-salesforce': minor
+---
+
+Modernize `query()` implementation using jsforce v3, adding support for
+additional query options [ `maxFetch`, `headers`, `responseTarget` and
```

Comment:
In your code, I see we are only using `maxFetch` and `autofetch`. Can one still pass the other options as mentioned here?

Reply:
Yes, you can pass the other options